tests/framework/Optimizers/optIteration.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/Optimizers.optIteration</name>
+    <author>ZHOUJ</author>
+    <created>2019-10-09</created>
+    <classesTested>Optimizer</classesTested>
```

Comment:
which one? the Conjugate

Reply:
``` <classesTested>Optimizer.ConjugateGradient</classesTested>```

Reply:
Is this connected with a requirement? It should be tagged with a requirement tag.

Reply:
```<requirements>R-RM-1</requirements>```